crates/services/upgradable-executor/src/executor.rs

```diff
@@ -115,10 +115,22 @@ impl<S, R> Executor<S, R> {
     /// on the block, native execution is used. If the version is not the same
     /// as in the block, then the WASM executor is used.
     pub const VERSION: u32 = StateTransitionBytecodeVersion::MIN;
+
     /// This constant is used along with the `version_check` test.
     /// To avoid automatic bumping during release, the constant uses `-` instead of `.`.
-    #[cfg(test)]
-    pub const CRATE_VERSION: &'static str = "0-26-0";
+    /// Each release should have its own new version of the executor.
```

Comment:
we should document this in the `contributing.md` file to explain that this needs to be manually updated every time the state transition function changes.